UserInterface/ProgressBarDemos/README.md

```diff
@@ -0,0 +1,23 @@
+---
+name: ProgressBar Demos
+description: This sample demonstrates how to use a ProgressBar in code and XAML with code-behind.
+topic: sample
+languages:
+- csharp
+products:
+- xamarin
+technologies:
+- xamarin-forms
+urlFragment: userinterface-progressbardemos
+---
+ProgressBar Demos
+==============
+
+This sample demonstrates how to use a `ProgressBar` in code and XAML with code-behind.
+
+For more information about this sample, see [Xamarin.Forms ProgressBar](https://docs.microsoft.com/xamarin/xamarin-forms/user-interface/progressbar).
+
```

Comment:
Add an image 🙂

Reply:
I added screenshots to the Screenshots folder following filename conventions from other samples. I hadn't added it to the readme itself because it seems like the gallery autoloads these. However, this sample is directly linked from the doc. So, I just added a reference to the most relevant Screenshots folder image.